pages/project/list/schema/index.js

```diff
@@ -5,6 +5,7 @@ module.exports = {
   },
   licenceHolder: {
     show: true,
+    sort: 'licenceHolder.lastName',
     accessor: 'licenceHolder.name',
```

Comment:
We might want to get rid of this anyway, and use a formatter to render the full name from the model instead on account of the fact that the `name` virtual property is flaky, and should be considered deprecated.

Reply:
yes agree, however the `sort` property is needed as we need to sort on lastName

Reply:
that was my way of saying "I'm staying in my lane", but actually I'm gunna swerve into oncoming traffic and fix this. Please hold.

Reply:
Shall I merge then?